docs/witx.md

```diff
@@ -22,8 +22,8 @@ the goals here are to remain aligned with interface types and other relevant
 WebAssembly standards and proposals wherever practical, and to be an input 
 into the design process of interface types.
 
-[module linking]: https://github.com/WebAssembly/module-linking/blob/master/proposals/module-linking/Explainer.md
-[interface types]: https://github.com/WebAssembly/interface-types/blob/master/proposals/interface-types/Explainer.md
+[module linking]: https://github.com/WebAssembly/module-linking/blob/main/design/proposals/module-linking/Explainer.md
+[interface types]: https://github.com/WebAssembly/interface-types/blob/main/proposals/interface-types/Explainer.md
```

Comment:
Technically this link isn't broken per se, because GitHub is smart enough to notice that `master` has been renamed to `main`, but we might as well correct it.